tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -133,6 +133,9 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-nocodec-bt-mtl-lbm.bin"
 "sof-hda-generic\;sof-hda-benchmark-eqfir16-tgl\;PLATFORM=TGL,HDA_CONFIG=benchmark,BENCH_CONFIG=eqfir16,BENCH_EQFIR_PARAMS=loudness"
 "sof-hda-generic\;sof-hda-benchmark-eqfir24-tgl\;PLATFORM=TGL,HDA_CONFIG=benchmark,BENCH_CONFIG=eqfir24,BENCH_EQFIR_PARAMS=loudness"
 "sof-hda-generic\;sof-hda-benchmark-eqfir32-tgl\;PLATFORM=TGL,HDA_CONFIG=benchmark,BENCH_CONFIG=eqfir32,BENCH_EQFIR_PARAMS=loudness"
+"sof-hda-generic\;sof-hda-benchmark-multiband_drc16-tgl\;PLATFORM=TGL,HDA_CONFIG=benchmark,BENCH_CONFIG=multiband16,BENCH_MULTIBAND_DRC_PARAMS=default"
```

Comment:
@marc-hb Is there a way to have a PLATFORM loop of TGL, MTL, etc. in cmake? Eventually all of this need to be duplicated to other platforms. Also the 16, 24, 32 format (and float possibly in the future) could be a loop if possible. These should replace sof/tools/test/topology and tools/build_tools/test/topology.

Reply:
> Is there a way to have a PLATFORM loop of TGL, MTL, etc. in cmake?

Yes it should be possible to have a `foreach` loop at the bottom of this file that appends more elements to $TPLGS after the initial, static definition. That should work.

Reply:
Thanks, there's now an effort to do it. Can you please check it if that's a good way to it.